app/views/preferences/index.html.erb

```diff
@@ -71,21 +71,21 @@
 </fieldset>
 <fieldset>
   <dl>
-    <dt><%= f.label :skin_id, ts('Your site skin') %> <%= link_to_help 'skins-basics' %>:</dt>
+    <dt><%= f.label :skin_id, ts('Your site skin') %> <%= link_to_help 'skins-basics' %></dt>
     <dd>
       <span class="actions"><%= link_to ts('Public Site Skins'), skins_path %></span>
       <%= f.select :skin_id, @available_skins.collect{|s| [s.title, s.id]} %>
     </dd>
 
-    <dt><%= f.label :time_zone, ts('Your time zone: ') %></dt>
+    <dt><%= f.label :time_zone, ts('Your time zone') %></dt>
     <dd><%= f.time_zone_select :time_zone, nil, :default => Time.zone.name %></dd>
     
     <% if Rails.env.test? || Rails.env.development? || $rollout.active?(:set_locale_preference, @user) %>
-      <dt><%= f.label :preferred_locale, ts('Your locale')%> <%= link_to_help 'locale-preferences' %></dt>
+      <dt><%= f.label :preferred_locale, ts('Your locale' )%> <%= link_to_help 'locale-preferences' %></dt>
```

Comment:
The space should be after the `)`, not before: `<dt><%= f.label :preferred_locale, ts('Your locale') %> <%= link_to_help 'locale-preferences' %></dt>`